src/main/java/callhub/connect/controllers/FileController.java

```diff
@@ -0,0 +1,55 @@
+package callhub.connect.controllers;
+import callhub.connect.data_access.DocumentRepository;
+import callhub.connect.data_access.LocalDataAccess;
+import callhub.connect.entities.FileDocument;
+import callhub.connect.entities.exceptions.FileLimitExceededException;
+import org.bson.types.Binary;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.io.File;
+import java.io.IOException;
+import java.time.LocalDate;
+
+@RestController
+@RequestMapping("/files")
+public class FileController {
+
+    @Autowired
+    DocumentRepository documentRepository;
+    @GetMapping("/upload")
+    public String uploadPDF () {
+        String currentDirectory = System.getProperty("user.dir");
```

Comment:
From my understanding this endpoint should take in the pdf to be uploaded as a parameter to send it to the MongoDB, or are we getting hardcoded files from within the project?